packages/sit-onyx/src/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Checks whether the given link is external (starts with http:// or https://).
+ */
+export const isExternalLink = (href: string): href is `http${"s" | ""}://${string}` => {
```

Comment:
Based on the comment below: 
```suggestion
export const hasCurrentOrigin = (href: string): href is `http${"s" | ""}://${string}` => {
```